go.mod

```diff
@@ -17,7 +17,7 @@ require (
 	github.com/iotexproject/go-fsm v1.0.0
 	github.com/iotexproject/go-p2p v0.2.12
 	github.com/iotexproject/go-pkgs v0.1.2-0.20200523040337-5f1d9ddaa8ee
-	github.com/iotexproject/iotex-address v0.2.1
+	github.com/iotexproject/iotex-address v0.2.2-0.20200718221015-84634386838e
```

Comment:
maybe tag a new version for this package?

Reply:
done v0.2.2